dbt_subprojects/solana/models/_sector/dex/dex_solana_base_trades.sql

```diff
@@ -14,6 +14,7 @@
 {% set solana_dexes = [
      ref('raydium_v3_base_trades')
    , ref('raydium_v4_base_trades')
+   , ref('raydium_v5_base_trades')
```

Comment:
fyi @JacobSharples we needed to add to this downstream model to then appear in `dex_solana.trades` 🙏